src/ui/molecules/stripe-elements.svelte

```diff
@@ -69,40 +70,11 @@
     }
   }
 
-  $: if (elements) {
-    (async () => {
-      const elementsConfiguration = gatewayParams.elements_configuration;
-      if (!elementsConfiguration) return;
-
-      await StripeService.updateElementsConfiguration(
-        elements,
-        elementsConfiguration,
-      );
-    })();
-  }
```

Comment:
It seems like this is a breaking change, we should call `updateElementsConfiguration` whenever `gatewayParams.elements_configuration` changes.

There is also no reason to call `updateElementsConfiguration` on mount.

Reply:
Perfect, understood now why we had this.

Since the `elements` variable will get assigned only by the initStripe function now, we don't need to watch the elements variables per se, but we need to update the Elements configuration whenever the `gatewayParams.elements_configuration` changes, adding it.